.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Community Discord Server (Login support requests go here!)
```

Comment:
"Community Discord Server (Login support requests go here!)"

Maybe rephrase that to something like
"Discord Community (Support for all server problems)"?

Reply:
i personally think its important to specifically call out login support requests because those are the most common kind we get here, tho i'm ofc fine with going with something else if there's a consensus around it

Reply:
I prefer it to sound more general, and not limited to login issues only.
"Discord Community (Find help with server issues like `Login`)"?
"Discord Community (Support for server issues: Account, Login...)"?
...